src/graphs/BasicGraphs.jl

```diff
@@ -346,6 +368,24 @@ function rem_vertices!(g::AbstractReflexiveGraph, vs; keep_edges::Bool=false)
   rem_parts!(g, :V, vs)
 end
 
+function convert(::Type{AbstractReflexiveGraph}, g::Graph)
+    X = ReflexiveGraph()
+    vs = add_parts!(X, :V, nv(g))
+    es = for e in edges(g)
+        let src = g[e, :src], tgt = g[e, :tgt]
```

Comment:
wow never seen this syntax before! is the idea that you want to use `src`/`tgt` as variable names and need this to avoid conflict with the surrounding namespace?